src/views/Pools/components/BountyCard.tsx

```diff
@@ -60,7 +61,18 @@ const BountyCard = () => {
     </>
   )
 
-  const [onPresentBountyModal] = useModal(<BountyModal TooltipComponent={TooltipComponent} />)
+  const [onPresentBountyModal] = useModal(
+    <BountyModal
+      estimatedCakeBountyReward={estimatedCakeBountyReward}
+      totalPendingCakeHarvest={totalPendingCakeHarvest}
+      callFee={callFee}
+      cakePriceBusd={cakePriceBusd}
+      TooltipComponent={TooltipComponent}
+    />,
+    true,
+    true,
+    'bountyModal',
```

Comment:
This is the missing part that cause refresh the other modal instances with this instance component. Thanks @chef-jojo